packages/restapi/src/lib/space/index.ts

```diff
@@ -14,3 +14,4 @@ export * from './approve';
 export * from './requests';
 export {spaceFeed as space} from './spaceFeed';
 export * from './Space'
+export * from './broadcastRaisedHand';
```

Comment:
@arn4b You dont have to export the broadcast raised hand function, we need to use it from the class object itself, just like how we call the join() method.

Reply:
ack, this was added to use the exported Interface, the function is being used from class object itself, but then the interface was not required.